src/components/services-page/ProductListing.tsx

```diff
@@ -0,0 +1,155 @@
+import { Link } from "gatsby";
+import React from "react";
+
+import decorativeCircle from "../../assets/images/decorative-circle.png";
+import "./css/product-listing.css";
+
+export const ProductListing = () => {
+  return (
+    <section className="product-listing">
+      <div className="product-listing__contents">
+        <img
+          alt=""
+          className="product-listing__decorative-circle"
+          role="presentation"
+          src={decorativeCircle}
+        />
+
+        <div className="product-listing__heading-container">
+          <h2 className="product-listing__heading">
+            Professional development services to meet your needs
+          </h2>
+          <div className="product-listing__icon-card-container">
+            <div className="product-listing__icon-card">
+              <img className="product-listing__icon" src="https://placehold.co/38x27" />
+              <h4 className="product-listing__icon-card-heading">Code Audit</h4>
+            </div>
+            <div className="product-listing__icon-card">
+              <img className="product-listing__icon" src="https://placehold.co/38x27" />
+              <h4 className="product-listing__icon-card-heading">Flight Plan</h4>
+            </div>
+            <div className="product-listing__icon-card">
+              <img className="product-listing__icon" src="https://placehold.co/38x27" />
+              <h4 className="product-listing__icon-card-heading">Staff Augmentation</h4>
+            </div>
+            <div className="product-listing__icon-card">
+              <img className="product-listing__icon" src="https://placehold.co/38x27" />
+              <h4 className="product-listing__icon-card-heading">Custom Development</h4>
+            </div>
+          </div>
+        </div>
+
+        <div className="product-listing__grid">
+          <div className="product-listing__image-container">
+            <img className="product-listing__image" src="https://placehold.co/585x438" />
+          </div>
+
+          <div className="product-listing__text-container">
+            <div className="product-listing__card">
+              <h4 className="product-listing__card-heading">Code Audit</h4>
```

Comment:
can we make these `<h3>` tags as we can't really skip heading numbers if we want to give the markup an "outline" structure